addons/account_payment/views/account_portal_templates.xml

```diff
@@ -25,7 +25,7 @@
                 <t t-if="invoice.state == 'posted' and last_tx.state == 'pending' and last_tx.acquirer_id.provider not in ('transfer', 'manual')">
                   <span class="badge badge-pill badge-warning"><span class="d-none d-md-inline"> Pending</span></span>
                 </t>
-                <t t-if="invoice.state == 'posted' and invoice.invoice_payment_state == 'paid' or last_tx.state == 'done'">
+                <t t-if="invoice.state == 'posted' and invoice.invoice_payment_state == 'paid'">
```

Comment:
No...
First of all, this break stable policy and if the problem is really present since v13 and only reported now, just fix it in master and the user for which it is problematic can edit his template directly in its own odoo instance.
Secondly, this hide the pill which is marked "paid" but if you open the invoice you still have a banner that say the same thing and which is still there.
Thirdly, this seems entirely wrong... I'm no expert in payment but from what I see when you select "wire transfer", the payment_transaction created behind the scene is directly in state=done. For other payment provider it might not be the case. Hence by just removing the condition last_tx.state == 'done' you might break other use case when we don't select "wire transfer" (there might be a use case where transaction has been registered and state is in done but not the invoice_payment_state)